README.md

```diff
@@ -1,8 +1,234 @@
+jsonrpc_api_http = "http://127.0.0.1:1235/rpc/v1"
+jsonrpc_api_ws = "wss://example.org/rpc/v0"
+auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.j94YYOr8_AWhGGHQd0q8JuQVuNhJA017SK9EUkqDOO0"
+accounts = ["t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq"]
+```
+
+> Beware: If you are already running the daemon, changes in the config file are only picked up after running `./bin/ipc_agent reload-config` so be sure to run it after editing your config.
+
+Finally, to test if the connection to the rootnet has been successful, we can for instance try to create a new wallet in the rootnet: 
+```
+./bin/ipc_agent wallet-new --key-type=bls --subnet=/root
+```
+
+### Running a subnet
+To spawn a new subnet, our IPC agent should be connected to at least the subnet of the parent we want to spawn the subnet from. You can refer to the previous section for information on how to run or connect to a rootnet. This instructions will assume the deployment of a subnet from `/root`, but the steps are equivalent for any other parent subnet. 
+
+#### Spawn subnet actor
+To run a subnet the first thing is to configure and create the subnet actor that will govern the subnet's operation:
+> TODO: Update instructions when the new IPC actor bundle is deployed as some of these parameters have changed.
+```bash
+./bin/ipc_agent create-subnet -p <parent-id> -n <subnet-name> --min-validator-stake 1 --min-valid
+ators 0 --finality-threshold 10 --check-period 10
+
+# Sample command execution
+./bin/ipc_agent create-subnet -p /root -n test --min-validator-stake 1 --min-valid
+ators 0 --finality-threshold 10 --check-period 10
+
+[2023-03-21T09:32:58Z INFO  ipc_agent::cli::commands::manager::create] created subnet actor with id: /root/t01002
+```
+This command deploys a subnet actor for a new subnet from the `root`, with a human-readable name `test`, that requires at least `1` validator to join the subnet to be able to mine new blocks, and with a checkpointing period to the parent of `10` blocks. We can see that the output of this command is the ID of the new subnet.
+
+#### Deploy subnet daemon
+Before joining a new subnet, our node for that subnet should be initialized, because as part of the joining process we would need to provide information about our validator network address, so other validators know how to dial them. For the deployment of subnet daemons we also provide a convenient infra script: 
+```bash
+$ ./bin/ipc-infra/run-subnet-docker.sh <lotus-api-port> <validator-libp2p-port> <subnet-id> <abssolute-path-validator-key>
+
+# Sample execution
+$ ./bin/ipc-infra/run-subnet-docker.sh 1239 1349 /root/t01002 /home/workspace/pl/lotus/scripts/ipc/src/wallet.key
+```
+> Beware: This script doesn't support the use of relative paths for the wallet path.
+
+The end of the log of the execution of this script provides a bit more of information than the previous one as it is implemented to be used for production deployments:
```

Comment:
I cannot get beyond this point probably because the container failed to build (I cannot see it running or even try to run the image in a container myself). (OS: Tuxedo OS, an Ubuntu fork) :
```
➜  ipc-agent git:(be8bb96) ./bin/ipc-infra/run-root-docker-1val.sh 1235 1379
[*] Running docker container for root in port 1235
[*] Waiting for the daemon to start
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: repo directory does not exist. Make sure your configuration is correct)
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
Not online yet... (could not get API info for FullNode: could not get api endpoint: API not running (no endpoint))
[...]
ERROR: timed out waiting for api to come online
```

Reply:
Not working for me. I'm now running on the largest lightsail instance (32gb ram, 8cpu) and the command returns quickly enough (<30s) but I still don't get any output beyond token and container 🤔

Reply:
Oh, OK! I know what could be, you may be passing the wallet key wrong. Happy to jump into a sync call if you are available to see if we can troubleshoot it (and add the corresponding checks when this happens).

Reply:
```
ubuntu@ip-172-26-28-130:~/ipc/ipc-agent$ docker ps
CONTAINER ID   IMAGE     COMMAND                  CREATED       STATUS       PORTS                                                                                                                          NAMES
ba55a6a40535   eudico    "/scripts/ipc/entryp…"   3 hours ago   Up 3 hours   1348/tcp, 1777/tcp, 2345/tcp, 3456/tcp, 0.0.0.0:1239->1234/tcp, :::1239->1234/tcp, 0.0.0.0:1349->1347/tcp, :::1349->1347/tcp   reverent_proskuriakova
0396b0dabc06   eudico    "/scripts/ipc/entryp…"   3 hours ago   Up 3 hours   1348/tcp, 1777/tcp, 2345/tcp, 3456/tcp, 0.0.0.0:1235->1234/tcp, :::1235->1234/tcp, 0.0.0.0:1379->1347/tcp, :::1379->1347/tcp   condescending_gates
ubuntu@ip-172-26-28-130:~/ipc/ipc-agent$ docker exec -it reverent_proskuriakova eudico mir validator config validator-addr
ERROR: validator not configured. Run `./eudico mir validator config init`
```

This is with the latest commit. If I'm supposed to pass a proper wallet, then I'm definitely doing that wrong: as far as I can tell we haven't yet exported a wallet key up to this point in the guide, so I'm just passing it an empty dir. I'll export the key and re-do, then :)

Reply:
I see you just swapped the order of the instructions. Working fine now, and the guide makes sense again :)

Never assume the reader has a brain ;)